code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1760,6 +1760,56 @@
 					H.update_inv_shoes(0)
 		M.clean_blood()
 
+/datum/reagent/space_cleaner/bleach
+	name = "Bleach"
+	id = BLEACH
+	description = "A strong cleaning compound. Corrosive and toxic when applied to soft tissue. Do not swallow."
+	reagent_state = LIQUID
+	color = "#FBFCFF" //rgb: 251, 252, 255
+
+/datum/reagent/space_cleaner/bleach/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	switch(data)
+		if(1 to 10)
+			M.adjustBruteLoss(3 * REM) //soft tissue damage
+		if(10 to INFINITY)
```

Comment:
yeah that's just the initial soft tissue damage, it should stop doing that after a bit because the bleach will just become more diluted with your stomach/bloodstream and therefore ineffective at dealing brute damage.
if having it last longer would make more sense then I can change that but that'll mean it's even more deadly